### ISSUE TITLE
Use authentication adapter of DoctrineModule instead of a custom one

### ISSUE THREAD

**Issue by froschdesign** [modir/laminas-book#3]
A custom authentication adapter is not needed if Doctrine is used:
https://modir.github.io/laminas-book/users/#writing-authentication-adapter

Use the adapter of DoctrineModule:
https://www.doctrine-project.org/projects/doctrine-module/en/6.1/authentication.html

**Comment by modir** [comment]
True, you are right this is the better option now.

TODO: Rewrite the chapter so that the DoctrineModule is used.

TO DECIDE: Is the current chapter about custom authentication adapter still relevant enough for other situations and should therefore be kept? (E.g as an appendix.)

**Comment by froschdesign** [comment]
> TO DECIDE: Is the current chapter about custom authentication adapter still relevant enough for other situations and should therefore be kept? (E.g as an appendix.)

Can be transferred to the official documentation because this part is only mentioned in the introduction to adapters.

**Comment by modir** [comment]
> Can be transferred to the official documentation because this part is only mentioned in the introduction to adapters.

Good idea. I didn't think of it. Would you merge it into https://docs.laminas.dev/laminas-authentication/adapter/intro/ or make a new page out of it? I am not sure what is better in this case.

**Comment by froschdesign** [comment]
> Would you merge it into https://docs.laminas.dev/laminas-authentication/adapter/intro/ or make a new page out of it?

New page, but let me do this, because you have enough to do here. :wink:
